src/client/swagger/managedFunctions.yml

```diff
@@ -167,57 +167,94 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/api/v2private/functions/{functionID}/runs':
+  '/api/v2/functions/{functionID}/invoke':
     get:
-      operationId: GetFunctionsIDRuns
+      operationId: GetFunctionsIDInvoke
       tags:
         - Functions
-      summary: List runs for a function
+      summary: Manually invoke a function with params in query
       parameters:
         - in: path
           name: functionID
           schema:
             type: string
           required: true
-          description: The ID of the function to get runs for.
+        - in: query
+          name: params
+          schema:
+            type: object
+            additionalProperties: true
+          style: form
+          explode: true
       responses:
         '200':
-          description: A list of function runs
+          description: Function invocation response
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/FunctionRunRecords'
+                $ref: '#/components/schemas/FunctionHTTPResponseData'
```

Comment:
Perhaps this should be `FunctionHTTPResponse` - as we're not sure if the user will return any data.

i.e. it may be totally fine to return 201 without data for a function that writes data to InfluxDB based on input provided to the function. Similarly, for 30x with redirects, it's totally fine to only return the Location header and no payload.